io-engine/src/grpc/v0/json_grpc.rs

```diff
@@ -36,9 +36,13 @@ impl JsonRpc for JsonRpcSvc {
     ) -> GrpcResult<JsonRpcReply> {
         let args = request.into_inner();
 
-        let result = self
+        let result = match self
```

Comment:
I don't understand, why did error conversion stop working?

Reply:
Ok, so it was throwing build error about 'From' trait not being able to convert here. But I just tried it again in original form and it builds, so most likely I must have had some conflicting toml files when I ran into that error. I'll revert this file's change.